.ci/scheduled-benchmark.groovy

```diff
@@ -119,3 +125,7 @@ pipeline {
     }
   }
 }
+
+def getVersion() {
+  return sh(script: """curl -s ${VERSION_FILE}  | grep defaultBeatVersion | cut -d'=' -f2 | sed 's#"##g'""", returnStdout: true).trim()
```

Comment:
probably needs an `-SNAPHOST` at the end, right? 

i'd also log out the version if doesn't happen already somewhere else

Reply:
aha, that's indeed a good point